assets/js/theme/product/image-gallery.js

```diff
@@ -41,14 +42,15 @@ export default class ImageGallery {
 
     selectNewImage(e) {
         e.preventDefault();
+        const first = 0;
         const $target = $(e.currentTarget);
         const imgObj = {
             mainImageUrl: $target.attr('data-image-gallery-new-image-url'),
             zoomImageUrl: $target.attr('data-image-gallery-zoom-image-url'),
             mainImageSrcset: $target.attr('data-image-gallery-new-image-srcset'),
             $selectedThumb: $target,
+            mainImageAlt: $target[first].children[first].alt,
         };
```

Comment:
I suppose that variable on line 45 is unnecessary here, because it has no calculation and no specific usage

Reply:
removed